examples/mpi/mpihello/slurm.sh

```diff
@@ -9,16 +9,15 @@ tar=$1
 imgdir=$2
 img=${2}/$(basename "${tar%.tar.gz}")
 
-if [[ -z $tar ]]; then
-    echo 'no tarball specified' 1>&2
+fatal () {
+    printf '%s\n\n' "$1" 2>&1
```

Comment:
I think the `slurm.sh` scripts should not be updated, since they are intended to be examples and ought to work outside Charliecloud.

Reply:
I did question whether this script should also be altered. The primary justification for doing updating this script is consistency. The new syntax is much sharper and easier read. FWIIW, the example script should still run outside of charliecloud. Thoughts?

Reply:
I think we should keep the examples simple, and it's simpler to have direct if statements rather than encapsulating them in another function for the user to think about. You might check with @heasterday. Do we promote this kind of error handling in user job scripts?